backend/spec/graphql/mutations/delete_journal_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::DeleteJournal do
+  let(:user) { create(:user) }
+  let!(:journal) { create(:journal, user:) }
```

Comment:
```suggestion
  let(:user) { create(:user) }
  let!(:journal) { create(:journal, user) }
```

ここってこうですかね？作成したuserを使いたいという気持ちかと思いました！

Reply:
気持ちとしては一緒なのですが、提案してもらったやつだと動かない疑惑です

おそらくこの前の省略の話と一緒で、新奥が今書いているやつは
`create(:journal, user: user)`
と同義なのかな

Reply:
↑今のrubocopの設定的にはこの書き方が正となるので、これでいく